URLQueryItemEncoder/URLQueryItemEncoder.swift

```diff
@@ -451,6 +448,3 @@ fileprivate class UnkeyedURLQueryItemReferencingEncoder: URLQueryItemReferencing
     referencedUnkeyedContainer.encodedItemsCount += items.count
   }
 }
-
-
-
```

Comment:
Please leave 1 or 2 empty line here. Just an a coding style of an old Objective-C :P

Reply:
but as I understand in Xcode 9 there is an extra space in the end of file already, do we need to add more extra.

anyway, I just put an empty space at the end of file so you can see new line in github PR 😬 .